common/logs.py

```diff
@@ -25,8 +27,8 @@
 # Disable this check since we have a bunch of non-constant globals in this file.
 # pylint: disable=invalid-name
 
-from common import retry
 from common import utils
+from common.retry import get_delay
```

Comment:
Annother annoying Google nit:we don't import functions, only modules:
(see https://google.github.io/styleguide/pyguide.html)

Reply:
Oh, got it!

I did this way to avoid logs importing retry and retry importing logs, not sure if it is a problem since log is not using retry.wrap decorator though, but I wanted to reuse the get_delay function

Reply:
THis is a good point. I think we should just reimplement get_delay here instead of having circular imports.

Reply:
Or actually, can you move it to a third module that both modules can import?